docs/source/en/optimization/xdit.md

```diff
@@ -0,0 +1,126 @@
+# xDiT
+
+xDiT is an inference engine designed for the parallel deployment of DiTs on large scale. xDiT provides a suite of efficient parallel approaches for Diffusion Models, as well as GPU kernel accelerations.
```

Comment:
Sounds super cool! I think a few more sentences about xDiT would really help users get a clearer and better idea of what it does 

```suggestion
[xDiT](https://github.com/xdit-project/xDiT) is an inference engine designed for the large scale parallel deployment of Diffusion Transformers (DiTs). xDiT provides a suite of efficient parallel approaches for Diffusion Models, as well as GPU kernel accelerations.
```